src/app/application.tsx

```diff
@@ -215,8 +209,20 @@ export class Application {
               this.config.CorsPolicy.TargetOrigins
             );
           }
+          if (this.config.CorsPolicy && this.config.CorsPolicy.Embedded) {
+            onSave({
```

Comment:
The `onSave` parameter is optional, but I don't see any checks before calling it here.